src/components/Shared/Loading/LoadingFullView.tsx

```diff
@@ -0,0 +1,39 @@
+import Loading from './Loading';
+
+import styled from 'styled-components';
+import { GlassWrapper } from '../containers/GlassWrapper';
+import { font_settings } from 'style/mixins';
+
+const Wrapper = styled(GlassWrapper)`
+  width: 340px;
```

Comment:
Czemu nie zrobiłaś tła na całą wysokość i szerokość przeglądarki? Żeby user nie mógł nic kliknąc?

Reply:
bo to wyłącza cały widok, i będzie brzydkie jak tak zrobię.

Reply:
no dobra. zostawmy to na razie, ale nie wykluczone, że będziemy tego później potrzebować